pkg/image/api/v1beta3/types.go

```diff
@@ -63,6 +63,8 @@ type NamedTagReference struct {
 	Name        string                `json:"name"`
 	Annotations map[string]string     `json:"annotations,omitempty"`
 	From        *kapi.ObjectReference `json:"from,omitempty"`
+	// Referenace states if the tag will be imported. Default value is false, which means the tag will be imported.
+	Reference bool `json:"reference,omitempty" description:"flag informing if tag is a reference (true) or is imported"`
```

Comment:
`if true consider this tag a reference only and do not attempt to import metadata about the image`